src/actions/contributorActions/index.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+import {
+  LOADED_CONTRIBUTOR,
+  LOADING_CONTRIBUTOR,
+  CONTRIBUTOR_ERROR,
+  
+} from "../types";
+import { GET_CONTRIBUTOR } from "../../constants";
+export const getContributors = () => async (dispatch) => {
```

Comment:
As far as async data is concerned, we can use Redux or React-Query to manage it. We use React-Query on our Jobs page and for posting Jobs as a researcher. I think it would be worth it to use React-Query to manage all our asynchronous data because it gives us caching and error/loading properties out of the box, and it would allow us to have a clean separation b/w our async data and everything else. 


For example, this could be a function `getContributors` in `src/lib/api/getContributors.js` that would like
```js
export const getContributors = async () => {
  const { data: { message } } = await axios.get({GET_CONTRIBUTOR})

  return message
}
```
and you use this function in `useQuery` from React-Query where it's needed. Another benefit is we wouldn't need the biolerplate code for the reducers and types and actions, just